mycli/main.py

```diff
@@ -428,10 +431,12 @@ def prompt_tokens(cli):
 
                 try:
                     logger.debug('sql: %r', document.text)
-                    if self.logfile:
+                    if self.logfile is not False:
```

Comment:
I would keep this as `if self.logfile`. That will catch any open file handles. Right now it will throw an error because a `self.logfile` with a value of `None` will be evaluated to `True`.